src/scripts/main.js

```diff
@@ -1,5 +1,42 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+
+document.querySelector('.wall').addEventListener('click', e => {
+  let topp = e.offsetY;
+  let left = e.offsetX;
+
+  if (e.target.classList.contains('spider')) {
+    if (parseFloat(spider.style.top) >= 375
+      || parseFloat(spider.style.top) <= 25
+      || parseFloat(spider.style.left) <= 25
+      || parseFloat(spider.style.left) >= 365) {
```

Comment:
calculate this `375, 25` values and move them into constants. code should work with any sizes of spider and wall